### DISCUSSION TITLE
How to import particles coordinates?

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#50] [Q&A] [answered]
How can I import particle coordinates? Also, if I do geometrical picking or template matching outside of nextPYP, and have also particle orientations, how can these be imported?  What is the nextPYP Euler angle convention for orientations?¨

edit: looks like this is answered in #37

**Comment by abartesaghi** [comment, marked as answer]
Just noting that this was answered in #37.
